sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_daily_summary_v1/query.sql

```diff
@@ -0,0 +1,21 @@
+SELECT
+  submission_date,
+  service,
+  country,
+  LANGUAGE,
```

Comment:
lowercase?

Reply:
I think that was BQ's format command doing this

Reply:
unfortunately, since that check is required and the check command complains about it, this will be capitlized